addons/website/static/src/snippets/s_faq_collapse/000.scss

```diff
@@ -33,3 +43,48 @@
         margin-bottom: 0;
     }
 }
+
+.s_faq_collapse_light {
+    .accordion .card {
+        border-left: 0px;
+        border-right: 0px;
+        .card-header {
+            background-color: transparent;
+        }
+    }
+}
+
+.s_faq_collapse_big {
+    .accordion .card {
+        border: 0px;
+        .card-header {
+            background-color: transparent;
+            font-size: 1.5rem;
```

Comment:
Try to parametrize instead of hardcoding values: use bootstrap variables or relative units so that the design adapts to the theme / user config. A font-size should probably be relative to the font-size variables, a margin/padding could use the "spacing" variables, etc etc (check your whole scss).

A designer might have a look afterwards but it's still a good idea to first try to do something great with non-hardcoded values :+1:

Reply:
I changed all absolute values that weren't already there.
That particular font-size now uses xx-large, I don't know if it still poses a problem.

Margin and padding values use btn-padding-xxxxx variables and should be OK.

Anyways, I fully expect a designer to make this fully awesome !